cloud/aws/templates/aws_oidc/main.tf

```diff
@@ -92,14 +93,36 @@ resource "aws_s3_bucket" "civiform_files_s3" {
   bucket = "civiform-files-s3"
 }
 
-resource "aws_s3_access_point" "civiform_files" {
+resource "aws_s3_bucket_public_access_block" "civiform_files_access" {
+  bucket                  = aws_s3_bucket.civiform_files_s3.id
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+resource "aws_s3_bucket_policy" "civiform_files_policy" {
   bucket = aws_s3_bucket.civiform_files_s3.id
-  name   = "civiform_files"
+  policy = data.aws_iam_policy_document.civiform_files_policy.json
+}
 
-  public_access_block_configuration {
-    block_public_acls       = true
-    block_public_policy     = true
-    ignore_public_acls      = true
-    restrict_public_buckets = true
+data "aws_iam_policy_document" "civiform_files_policy" {
+  statement {
+    actions   = ["s3:*"]
+    effect    = "Deny"
+    resources = ["${aws_s3_bucket.civiform_files_s3.arn}/*"]
+    not_principals {
+      type        = "AWS"
+      identifiers = [aws_iam_role.apprunner_instance_role.arn]
+    }
+  }
+  statement {
+    actions   = ["s3:GetObject", "s3:PutObject"]
```

Comment:
ooo is this like you can get but you need to have authenticated first?

Reply:
yup. only the apprunner job should have access to this. Seattle does it in a similar way.